airbyte-integrations/connectors/source-trustpilot/main.py

```diff
@@ -2,7 +2,11 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
-from source_trustpilot.run import run
+import sys
+
+from airbyte_cdk.entrypoint import launch
+from source_trustpilot import SourceTrustpilot
 
 if __name__ == "__main__":
-    run()
+    source = SourceTrustpilot()
```

Comment:
The change to run the source in `run.py` instead of `main.py` was intentional, and we should keep it. The trick here is that `run.py` is part of the package source, and when the source is installed with pip in `PyAirbyte`, we use it to run the source directly without Docker environment.

Reply:
Done!